RecoJets/JetProducers/interface/ECFAdder.h

```diff
@@ -11,12 +11,14 @@
 #include "DataFormats/Common/interface/ValueMap.h"
 #include "fastjet/contrib/EnergyCorrelator.hh"
 #include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
 
 class ECFAdder : public edm::stream::EDProducer<> {
 public:
   explicit ECFAdder(const edm::ParameterSet& iConfig);
 
   void produce(edm::Event& iEvent, const edm::EventSetup& iSetup) override;
+  void addFJParticle(std::vector<fastjet::PseudoJet>& FJparticles, const reco::CandidatePtr& dp) const;
```

Comment:
FJparticles
start local variables and data members with lower case letter. Capitalization is reserved for class names.